File: android/app/src/main/java/com/texasgamer/zephyr/util/DimenUtils.java
package com.texasgamer.zephyr.util;

import android.content.res.Resources;

import androidx.annotation.Dimension;
import androidx.annotation.Px;

public class DimenUtils {

    @Px
    public static int dpToPx(@Dimension(unit = Dimension.DP) int dp) {
        return (int) (dp * Resources.getSystem().getDisplayMetrics().density);
    }

    @Dimension(unit = Dimension.DP)
    public static int pxToDp(@Px int px) {
        return (int) (px / Resources.getSystem().getDisplayMetrics().density);
    }
}

File: android/app/src/main/java/com/texasgamer/zephyr/util/DimenUtils.kt
package com.texasgamer.zephyr.util

import android.content.res.Resources
import androidx.annotation.Dimension
import androidx.annotation.Px

/**
 * Dimension utilities.
 */
object DimenUtils {
    @JvmStatic
    @Px
    fun dpToPx(@Dimension(unit = Dimension.DP) dp: Int): Int {
        return (dp * Resources.getSystem().displayMetrics.density).toInt()
    }

    @Dimension(unit = Dimension.DP)
    fun pxToDp(@Px px: Int): Int {
        return (px / Resources.getSystem().displayMetrics.density).toInt()
    }
}